app/main.py

```diff
@@ -1,8 +1,7 @@
 from fastapi import FastAPI
 import psycopg2
-from . import models
 from .database import engine
-from .routers import user,post
+from .routers import user,post,auth
```

Comment:
In general, try to avoid using relative imports - that's a possible way to the import problems in the future. 
It's not critical, just a recommendation.